servicecontrol/servicecontrol-instances/distributed-instances.md

```diff
@@ -0,0 +1,217 @@
+
+4. Start ServiceControl master.
+5. Validate ServiceInsight and ServicePulse are connecting to the master instance only.
+
+### Multi-instance installation with split audit queue
+
+This section walks through a fresh installation of multiple instances of ServiceControl with split audit queues. 
+
+```mermaid
+graph TD
+SI[ServiceInsight] -. connected to .->Master
+SP[ServicePulse] -. connected to .-> Master
+Endpoints -- audits to--> AuditsMain[audit-master]
+OtherEndpoints -- audits to--> AuditsSlave(audit-slave)
+Endpoints -- errors to --> Errors(error)
+OtherEndpoints -- errors to --> Errors(error)
+Master[ServiceControl<br/>Master]
+Slave[ServiceControl<br/>Slave]
+AuditsMain -- ingested by --> Master
+Errors-- ingested by --> Master
+AuditsSlave -- ingested by --> Slave
+Master -. connected to .-> Slave
+```
+
+As an example two ServiceControl instances are used where one is the master and the other one the slave for auditing. Some of the endpoints will forward their audit messages to `audit-master` while some of the endpoints will forward their audit messages to `audit-slave`. All error messages are forwared to `error` queue. ServiceInsight and ServicePulse are connected to the master instance.
+
+1. Install the ServiceControl master instance which ingests messages from the `audit-master` queue according to the [installation guidelines](/servicecontrol/installation.md).
+2. Install the ServiceControl slave instance (on separate infrastructure) which ingests messages from the `audit-slave` queue according to the [installation guidelines](/servicecontrol/installation.md). The ServiceControl instance name and the port are required to configure the master instance. In this example, the name `Particular.ServiceControl.Slave` and the port `33334` is used. Make sure error queue processing is disabled by specifying `!disable` as the error queue field in the ServiceControl Management Utility, or as the error queue parameter of the [Powershell](/servicecontrol/installation-powershell.md) installation script, or by editing `ServiceControl.exe.config` as shown below:
+
+```xml
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+    <appSettings>
+        <add key="ServiceBus/ErrorQueue" value="!disabled" />
+    </appSettings>
+</configuration>
+```
+
+If the error queue is set to `!disable` then Error forwarding will be ignored even if enabled.
+
+3. Stop the ServiceControl master instance and edit the [`ServiceControl.exe.config` ](/servicecontrol/creating-config-file.md) with the `RemoteInstances` key. The value for the key is a json array.
+
+```xml
+<configuration>
+  <appSettings>
+    <add key="ServiceControl/RemoteInstances" value="[{'api_uri':'http://localhost:33334/api', 'queue_address':'Particular.ServiceControl.Slave'}]'"/>
+  </appSettings>/
+</configuration>
+```
+
+4. Start ServiceControl master.
+5. Validate ServiceInsight and ServicePulse are connecting to the master instance only.
+
+### Splitting an existing installation
```

Comment:
This section needs quite a bit more fleshing out

Reply:
Can you be a bit more specific? 😀

Reply:
Suggest to merge the PR and "polish" it later if needed. Would that work?